server.js

```diff
@@ -17,14 +17,13 @@ app.use(bodyParser.json());
 // Parse application/x-www-form-urlencoded
 app.use(bodyParser.urlencoded({extended: false}));
 
-app.use(stormpath.init(app, {web: {produces: ['application/json']}}));
+app.use(stormpath.init(app, {website: true}));
```

Comment:
@a-ashbeck after changing this I was able to go to `/register` register and then login